app/models/schedule.rb

```diff
@@ -1,7 +1,6 @@
 class Schedule < ActiveRecord::Base
   include RankedModel
 
-  has_many :volunteers, through: :schedule_volunteers
```

Comment:
I'm not sure - if they are instances of `Schedule` then it will fail:

```
[1] pry(main)> Schedule.new.volunteers
ActiveRecord::HasManyThroughAssociationNotFoundError: Could not find the association :schedule_volunteers in model Schedule
from /Users/ryanfoster/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/activerecord-3.2.22.2/lib/active_record/reflection.rb:501:in `check_validity!'
```